src/Octokit.Webhooks.AspNetCore/GitHubWebhookOptions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Octokit.Webhooks.AspNetCore;
+
+public sealed record GitHubWebhookOptions
```

Comment:
Should this be sealed? I can imagine a use case (at least personally) where I would like to be able to also lump unrelated configuration to webhooks (like my client secret or app id) and not have to create a separate class for webhooks and GitHub app configurations.

Reply:
I sealed it mostly because I seal all of the classes in this library. The performance benefits of the webhook object classes is probably more important than this object. Webhook objects will be constructed on each event, whereas this will be constructed once, on startup.

I can send a follow-up PR to unseal it.